app/uk/gov/hmrc/helptosavestub/controllers/DESThresholdController.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavestub.controllers
+
+import com.google.inject.Inject
+import play.api.libs.json.{JsValue, Json}
+import play.api.mvc.{Action, AnyContent}
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.helptosavestub.config.AppConfig
+import uk.gov.hmrc.helptosavestub.util.Logging
+import uk.gov.hmrc.play.bootstrap.controller.BaseController
+
+class DESThresholdController @Inject() (implicit override val runModeConfiguration: Configuration,
+                                        override val environment: Environment) extends AppConfig(runModeConfiguration, environment)
+  with BaseController with DESController with Logging {
+
+  val thresholdAmount: JsValue =
+    Json.parse("""
+                 |{
+                 |  "thresholdAmount" : "100"
```

Comment:
this value should be read from config rather than being hard coded

Reply:
you're right, thanks